Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -772,31 +772,21 @@ int CLuaEngineDefs::EngineSetModelLODDistance(lua_State* luaVM)
     return 1;
 }
 
-int CLuaEngineDefs::EngineResetModelLODDistance(lua_State* luaVM)
+//  bool engineResetModelLODDistance(int / string modelID)
+bool CLuaEngineDefs::EngineResetModelLODDistance(const std::variant<const std::string, const unsigned short> variantModelID)
 {
-    SString          strModel = "";
-    CScriptArgReader argStream(luaVM);
-    argStream.ReadString(strModel);
+    const unsigned short modelID = std::holds_alternative<std::string>(variantModelID) ? CModelNames::ResolveModelID(std::get<std::string>(variantModelID)) : std::get<unsigned int>(variantModelID);
```

Comment:
EngineResetModelLODDistance

```suggestion
    unsigned short modelID;
    if (std::holds_alternative<std::string>(variantModelID))
        modelID = CModelNames::ResolveModelID(std::get<std::string>(variantModelID));
    else
        modelID = std::get<unsigned int>(variantModelID);
```